features/display_menu_item.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Visitor can see a prodcut on the menu
+    As a visitor,
+    In order to get an overview of the restaurant's products,
+    I would like to be able to see a product on the menu.
+
+    Background:
+        Given the following menu item exist
+            | name         | description                                                                             | price   | animal |
+            | Kitty Heaven | Fresh, seared yellow tail tuna, served on a bed of catnip picked daily from our garden. | 140     | cat    |
```

Comment:
I would like to see multiple MenuItems in this feature

Reply:
Two products are now in the background for the feature test.